tasks/species_typing/task_staphopiasccmec.wdl

```diff
@@ -9,26 +9,48 @@ task staphopiasccmec {
     String samplename
     String docker = "quay.io/biocontainers/staphopia-sccmec:1.0.0--hdfd78af_0"
     Int disk_size = 100
-    Int? cpu = 2
-
-    # Parameters
-    # --hamming Report the results as hamming distances
-    Boolean hamming = false
+    Int cpu = 1
   }
   command <<<
+    # get version
     staphopia-sccmec --version 2>&1 | sed 's/^.*staphopia-sccmec //' | tee VERSION
+
+    # run staphopia-sccmec on input assembly; hamming option OFF; outputs are true/false
     staphopia-sccmec \
-      ~{true="--hamming" false="" hamming} \
-      --assembly ~{assembly} > ~{samplename}.tsv
+      --assembly ~{assembly} > ~{samplename}.staphopia-sccmec.summary.tsv
+
+    # run staphopia-sccmec on input assembly; hamming option ON; outputs are the hamming distance; 0 is exact match
```

Comment:
A bit confused here. Looks like it the task was originally written to take a boolean to invoke the `--hamming` flag, but now things are written to run staphopia-sccmec twice: once without this flag, and once with it. 

Any reason it shouldn't remain a boolean input?

Reply:
You are correct, the task runs the tool twice, once with the `--hamming` flag and once without. I figured that it would be useful to always have both output TSVs available to the user instead of just one (based on the old boolean input that I removed).

the tool runs very quick so it's not too much extra time to run a second time

Reply:
I don't believe there is an option to run the tool once and allow for outputting of both TSVs, so I opted to just run it twice

Reply:
Ah, okay. Thanks for clarifying. And I agree: that hamming distance output will be really helpful when laboratories want to take a closer look at things.